2016-06-09T14:10:09.186Z - info: Admin GUI listening on port 8888
2016-06-09T15:05:21.020Z - info: Admin GUI listening on port 8888
2016-06-09T15:05:21.495Z - info: Created system-admin user
2016-06-09T16:06:51.116Z - error: uncaughtException: MongoDB - Error: failed to connect to [localhost:27017] date=Thu Jun 09 2016 12:06:51 GMT-0400 (EDT), pid=16449, uid=502, gid=20, cwd=/Users/cartercook/projects/mikbot/server/src, execPath=/usr/local/Cellar/node/6.2.0/bin/node, version=v6.2.0, argv=[/usr/local/Cellar/node/6.2.0/bin/node, /Users/cartercook/projects/mikbot/server/src/admin], rss=56967168, heapTotal=38793216, heapUsed=30481864, loadavg=[1.875, 1.82421875, 1.810546875], uptime=183354, trace=[column=9, file=/Users/cartercook/projects/mikbot/server/src/admin.js, function=, line=35, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=NativeConnection.emit, line=188, method=emit, native=false, column=10, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js, function=, line=147, method=null, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Db.emit, line=191, method=emit, native=false, column=19, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=emit, line=254, method=null, native=false, column=17, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=Server.Base._emitAcrossAllDbInstances, line=504, method=Base._emitAcrossAllDbInstances, native=false, column=14, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js, function=.<anonymous>, line=568, method=null, native=false, column=13, file=events.js, function=emitThree, line=116, method=null, native=false], stack=[Error: MongoDB - Error: failed to connect to [localhost:27017],     at NativeConnection.<anonymous> (/Users/cartercook/projects/mikbot/server/src/admin.js:35:9),     at emitOne (events.js:96:13),     at NativeConnection.emit (events.js:188:7),     at Db.<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:147:10),     at emitTwo (events.js:106:13),     at Db.emit (events.js:191:7),     at emit (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:254:19),     at Server.Base._emitAcrossAllDbInstances (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:504:17),     at .<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js:568:14),     at emitThree (events.js:116:13)]
2016-06-09T16:06:52.121Z - error: uncaughtException: MongoDB - Error: failed to connect to [localhost:27017] date=Thu Jun 09 2016 12:06:52 GMT-0400 (EDT), pid=16449, uid=502, gid=20, cwd=/Users/cartercook/projects/mikbot/server/src, execPath=/usr/local/Cellar/node/6.2.0/bin/node, version=v6.2.0, argv=[/usr/local/Cellar/node/6.2.0/bin/node, /Users/cartercook/projects/mikbot/server/src/admin], rss=57221120, heapTotal=37744640, heapUsed=30184248, loadavg=[1.875, 1.82421875, 1.810546875], uptime=183355, trace=[column=9, file=/Users/cartercook/projects/mikbot/server/src/admin.js, function=, line=35, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=NativeConnection.emit, line=188, method=emit, native=false, column=10, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js, function=, line=147, method=null, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Db.emit, line=191, method=emit, native=false, column=19, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=emit, line=254, method=null, native=false, column=17, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=Server.Base._emitAcrossAllDbInstances, line=504, method=Base._emitAcrossAllDbInstances, native=false, column=14, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js, function=.<anonymous>, line=568, method=null, native=false, column=13, file=events.js, function=emitThree, line=116, method=null, native=false], stack=[Error: MongoDB - Error: failed to connect to [localhost:27017],     at NativeConnection.<anonymous> (/Users/cartercook/projects/mikbot/server/src/admin.js:35:9),     at emitOne (events.js:96:13),     at NativeConnection.emit (events.js:188:7),     at Db.<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:147:10),     at emitTwo (events.js:106:13),     at Db.emit (events.js:191:7),     at emit (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:254:19),     at Server.Base._emitAcrossAllDbInstances (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:504:17),     at .<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js:568:14),     at emitThree (events.js:116:13)]
2016-06-09T16:06:53.129Z - error: uncaughtException: MongoDB - Error: failed to connect to [localhost:27017] date=Thu Jun 09 2016 12:06:53 GMT-0400 (EDT), pid=16449, uid=502, gid=20, cwd=/Users/cartercook/projects/mikbot/server/src, execPath=/usr/local/Cellar/node/6.2.0/bin/node, version=v6.2.0, argv=[/usr/local/Cellar/node/6.2.0/bin/node, /Users/cartercook/projects/mikbot/server/src/admin], rss=57253888, heapTotal=37744640, heapUsed=30404464, loadavg=[1.8046875, 1.81005859375, 1.8056640625], uptime=183356, trace=[column=9, file=/Users/cartercook/projects/mikbot/server/src/admin.js, function=, line=35, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=NativeConnection.emit, line=188, method=emit, native=false, column=10, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js, function=, line=147, method=null, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Db.emit, line=191, method=emit, native=false, column=19, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=emit, line=254, method=null, native=false, column=17, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js, function=Server.Base._emitAcrossAllDbInstances, line=504, method=Base._emitAcrossAllDbInstances, native=false, column=14, file=/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js, function=.<anonymous>, line=568, method=null, native=false, column=13, file=events.js, function=emitThree, line=116, method=null, native=false], stack=[Error: MongoDB - Error: failed to connect to [localhost:27017],     at NativeConnection.<anonymous> (/Users/cartercook/projects/mikbot/server/src/admin.js:35:9),     at emitOne (events.js:96:13),     at NativeConnection.emit (events.js:188:7),     at Db.<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:147:10),     at emitTwo (events.js:106:13),     at Db.emit (events.js:191:7),     at emit (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:254:19),     at Server.Base._emitAcrossAllDbInstances (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/base.js:504:17),     at .<anonymous> (/Users/cartercook/projects/mikbot/server/src/node_modules/mongoose/node_modules/mongodb/lib/mongodb/connection/server.js:568:14),     at emitThree (events.js:116:13)]
2016-06-09T16:09:09.204Z - error: uncaughtException: controls is not defined date=Thu Jun 09 2016 12:09:09 GMT-0400 (EDT), pid=16543, uid=502, gid=20, cwd=/Users/cartercook/projects/mikbot/server/src, execPath=/usr/local/Cellar/node/6.2.0/bin/node, version=v6.2.0, argv=[/usr/local/Cellar/node/6.2.0/bin/node, /Users/cartercook/projects/mikbot/server/src/admin], rss=41496576, heapTotal=32501760, heapUsed=20357120, loadavg=[1.76318359375, 1.82568359375, 1.81201171875], uptime=183492, trace=[column=43, file=/Users/cartercook/projects/mikbot/server/src/admin.js, function=, line=107, method=null, native=false, column=32, file=module.js, function=Module._compile, line=541, method=_compile, native=false, column=10, file=module.js, function=Object.Module._extensions..js, line=550, method=Module._extensions..js, native=false, column=32, file=module.js, function=Module.load, line=458, method=load, native=false, column=12, file=module.js, function=tryModuleLoad, line=417, method=null, native=false, column=3, file=module.js, function=Function.Module._load, line=409, method=Module._load, native=false, column=10, file=module.js, function=Function.Module.runMain, line=575, method=Module.runMain, native=false, column=18, file=node.js, function=startup, line=160, method=null, native=false, column=3, file=node.js, function=null, line=449, method=null, native=false], stack=[ReferenceError: controls is not defined,     at Object.<anonymous> (/Users/cartercook/projects/mikbot/server/src/admin.js:107:43),     at Module._compile (module.js:541:32),     at Object.Module._extensions..js (module.js:550:10),     at Module.load (module.js:458:32),     at tryModuleLoad (module.js:417:12),     at Function.Module._load (module.js:409:3),     at Function.Module.runMain (module.js:575:10),     at startup (node.js:160:18),     at node.js:449:3]
2016-06-09T16:12:20.802Z - info: Admin GUI listening on port 8888
2016-06-09T16:43:55.531Z - info: Admin GUI listening on port 8888
